src/utils/tableData.tsx

```diff
@@ -28,18 +28,29 @@ function TableData({
   page,
   prepareRow,
   tableDataCondition,
-  fromPage
+  fromPage,
+  setDagRunId,
+  selectedDagIndex
 }: any) {
-  const [selectedRowIndex, setSelectedRowIndex] = useState(-1);
+  const [selectedRowIndex, setSelectedRowIndex] = useState(selectedDagIndex);
 
   const displayData = page ? page : rows;
 
-  const handleRowClicked = (row: Row, index: number) => {
+  const handleRowClicked = (row: any, index: number) => {
```

Comment:
Add type

Reply:
We have removed this part of code in upcoming PR (Row click functionality not needed)